numba_pandas/tests/test_pandas.py

```diff
@@ -0,0 +1,24 @@
+import numba
+import pandas
+
+from packaging import version
+
+# numba <0.46 requires explicit import extension
+if version.parse(numba.__version__) < version.parse('0.46'):
+    from numba_pandas import _init_extension
+    _init_extension()
+
+
+def test_function():
+    pandas_func = pandas.Series
+
+    @numba.njit
```

Comment:
I don't think we need additional test suite for this functionality. It will be the only functionality we have.
Could you please chenge several `hpat.jit` dicorators in our regular test system? For example, in `test_series.py` with several tests which are already support `new style` functionality?

Reply:
I will create separate PR.

Reply:
#256